bootstrap/README.md

```diff
@@ -106,3 +125,170 @@ cargo run --release --example compute_hashes -- --file ./output/canister_state.b
 ```
 
 The hashes of each chunk are saved in `./bootstrap/chunk_hashes.txt` and can be used later when building the `uploader` canister in Docker.
+
+## 6. Build Canisters
+
+```shell
+# Go back to root repo directory
+$ cd ..
+
+# Build all, specifying the path to chunk_hashes.txt
+$ docker build --build-arg CHUNK_HASHES_PATH=/bootstrap/chunk_hashes.txt  -t canisters .
+
+# Extract canister's WASM
+$ docker run --rm --entrypoint cat canisters /uploader.wasm.gz > uploader.wasm.gz
+$ docker run --rm --entrypoint cat canisters /ic-btc-canister.wasm.gz > ic-btc-canister.wasm.gz
+
+# Verify SHA-256 of the canister's WASM.
+$ sha256sum *.wasm.gz
+c6abf3605cd33d0d640a648ecc1aaf33999032775436481485468a75024f38bc  ic-btc-canister.wasm.gz
+2f9a1f7ee91ce2e2c29cc78040197b2687c25ac7fd76a609c79a72c67e3ca1d8  uploader.wasm.gz
+```
+
+## 7. Install Uploader Canister & Upload Chunks
+
+Prepare install arguments
+```shell
+# Get canister state size
+$ ls -al ./bootstrap/output/canister_state.bin
+-rw-rw-r-- 1 maksym maksym 1149304832 Jan 13 11:46 ./bootstrap/output/canister_state.bin
+```
+
+Calculate required number of pages, page is `64 * 1024` bytes
+```txt
+1149304832 / (64 * 1024) = 17537
+```
+
+Calculate args hash
+```shell
+$ didc encode -t '(nat64)' "(17537)" | xxd -r -p | sha256sum
+e299fbe18558a3646ab33e5d28eec04e474339f235cf4f22dd452c98f831a249  -
+```
+
+```shell
+TESTNET_BITCOIN_CANISTER_ID="g4xu7-jiaaa-aaaan-aaaaq-cai"; \
+    TESTNET_WATCHDOG_CANISTER_ID="gjqfs-iaaaa-aaaan-aaada-cai"; \
+    MAINNET_BITCOIN_CANISTER_ID="ghsi2-tqaaa-aaaan-aaaca-cai"; \
+    MAINNET_WATCHDOG_CANISTER_ID="gatoo-6iaaa-aaaan-aaacq-cai"
```

Comment:
nit: just list the commands one by one?

Reply:
it feels easier to have all canister ids together (to quickly verify those are correct) and only manipulate descriptive variable names, vs ids. I'll keep it as is.